### DISCUSSION TITLE
Best QR Settings for Motion GoPro Hero 7

### DISCUSSION THREAD

**Discussion by Rick5472** [gopro/labs#412] [Q&A]
What are the best QR settings for motion detection recording on a GoPro Hero 7 for observing a bird feeder? Has someone come up with the best settings, and if so, can post them?

**Comment by dnewman-gpsw** [comment]
Each scenario is different.  You will need to determine the sensitivity for your environment, lighting, and the amount of unrelated motion in the frame.

**Reply by Rick5472** [reply]
How about  "ballpark" settings to get started?

**Reply by dnewman-gpsw** [reply]
Try the defaults: https://gopro.github.io/labs/control/motion/

**Reply by Rick5472** [reply]
Thanks! I’ll check it out.
